### PR TITLE
Update ome-faq.yml

### PR DESCRIPTION
**Pull request by samschan-ms**
Added portal log FAQ. @KCCross , @v-tophillips please review

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/compliance/ome-faq.yml [hunk 1]
```diff
@@ -169,9 +169,9 @@ sections:
           Outlook on the web supports revocation of protected mail.  See [How to revoke an encrypted message that you sent](revoke-ome-encrypted-mail.md#how-to-revoke-an-encrypted-message-that-you-sent) for details.
 
       - question: |
-          Does the OME portal support preview of encrypted attachments in protected emails?
+          Does the encrypted message portal support preview of encrypted attachments in protected emails?
         answer: |
-          The OME portal support preview of any encrypted attachment copies added to the encrypted mail. The support file types includes Word, Excel, Powerpoint, and PDF files.  
+          The encrypted message portal support preview of any encrypted attachment copies added to the encrypted mail. The support file types includes Word, Excel, Powerpoint, and PDF files.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.yml]:
```suggestion
          The encrypted message portal support preview of any encrypted attachment copies added to the encrypted mail. The support file types include Word, Excel, Powerpoint, and PDF files.  
```

File: microsoft-365/compliance/ome-faq.yml [hunk 2]
```diff
@@ -192,7 +192,12 @@ sections:
           Can I customize encrypted messages with my company branding?
         answer: |
           Yes, for mail sent from an Exchange Online mailbox in your organization! For information on customizing email messages and the OME portal, see [Add your organization's brand to your encrypted messages](add-your-organization-brand-to-encrypted-messages.md).
-            
+
+      - question: |
+          What types of recipients do the encrypted message portal activity logs work?
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.yml]:
```suggestion
          On what types of recipients do the encrypted message portal activity logs work?
```

File: microsoft-365/compliance/ome-faq.yml [hunk 3]
```diff
@@ -192,7 +192,12 @@ sections:
...
+        answer: |
+          The encrypted message portal activity logs only capture events for external recipients by accessing the encrypted message portals.  Any activities in email clients triggered by external recipients are not recorded. For internal recipients, see [Purview Audit (Premium) - Mail items accessed logs](compliance/mailitemsaccessed-forensics-investigations#the-mailitemsaccessed-mailbox-auditing-action).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.yml]:
```suggestion
          The encrypted message portal activity logs only capture events for external recipients by accessing the encrypted message portals.  Any activities in email clients triggered by external recipients are not recorded. For internal recipients, see [Purview Audit (Premium) - Mail items accessed logs](/compliance/mailitemsaccessed-forensics-investigations#the-mailitemsaccessed-mailbox-auditing-action). 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @samschan-ms, kindly commit the suggested changes.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Comment by KCCross** [comment]
I reworked these changes through the private repo as some needed additional clarification.

**Pull request closed without merging**